Description: Remove -m MODE from rapepr help
Origin: commit:430a21084665da35a715e9055d72a13487972969
Author: Dave Beckett <dave@dajobe.org>
Last-Update: 2012-09-07

    Remove -m MODE from help
    
    This option was removed in commit f94fa561db05b21132b14a2b72f05b77e666c252
    on Wed Apr 28 21:31:54 2010 -0700 as part of the Raptor V2 work.

diff --git a/utils/rapper.c b/utils/rapper.c
index c130177..31affb8 100644
--- a/utils/rapper.c
+++ b/utils/rapper.c
@@ -707,7 +707,6 @@ main(int argc, char *argv[])
     puts(HELP_TEXT("f OPTION(=VALUE)", "feature OPTION(=VALUE)", HELP_PAD "Set parser or serializer options" HELP_PAD "Use `-f help' for a list of valid options"));
     puts(HELP_TEXT("g", "guess           ", "Guess the input syntax (same as -i guess)"));
     puts(HELP_TEXT("h", "help            ", "Print this help, then exit"));
-    puts(HELP_TEXT("m MODE", "mode MODE  ", "Set parser mode - 'lax' (default) or 'strict'"));
     puts(HELP_TEXT("q", "quiet           ", "No extra information messages"));
     puts(HELP_TEXT("r", "replace-newlines", "Replace newlines with spaces in literals"));
 #ifdef SHOW_GRAPHS_FLAG
